solidity/test/isms/legacyMultisigIsm.test.ts

```diff
@@ -57,7 +63,7 @@ describe('LegacyMultisigIsm', async () => {
   describe('#moduleType', () => {
     it('returns the correct type', async () => {
       expect(await multisigIsm.moduleType()).to.equal(
-        types.InterchainSecurityModuleType.MULTISIG,
```

Comment:
Directly exporting the contents of the utils package has 3 advantages:
1. Nicer DevX in editors
2. Allows for tree-shaking by bundlers
3. Avoids problem of people intentionally importing like this `@hyperlane-xyz/utils/dist/foo/bar/`